documentation/modules/oauth/proc-oauth-authorization-broker-config.adoc

```diff
@@ -69,6 +69,8 @@ spec:
 ----
 <1> Type `keycloak` enables Keycloak authorization.
 <2> URI of the Keycloak token endpoint. For production, always use HTTPs.
+When you configure token-based `oauth` authentication, you specify a `jwksEndpointUri` as the URI for local JWT validation.
+The hostname for the `tokenEndpointUri` URI must be the same.
```

Comment:
@scholzj @mstruk The _Configuring OAuth 2.0 authorization support_ procedure mentions this in the prereqs: _You use the same Keycloak server endpoint when you set up authorization._. But it's not very obvious. So I added this to the  _tokenEndpointUri_ callout